ci/packaging/suse/obsfiles_maker.sh

```diff
@@ -19,14 +17,16 @@ trap clean ERR
 rm -rf "${obs_files}"
 mkdir -p "${obs_files}"
 
-cp "${image}/${image}.kiwi" "${tmp_dir}"
-[ -f "${image}/config.sh"  ] && cp "${image}/config.sh" "${tmp_dir}"
-[ -d "${image}/root"  ] && tar -caf "${tmp_dir}/root.tar.gz" \
-    -C "${image}/root" .
-[ -f "${image}/_service"  ] && cp "${image}/_service" "${tmp_dir}"
-make CHANGES="${tmp_dir}/${image}.changes.append" suse-changelog
+install -p -m 644 "${image}/${image}.kiwi" "${obs_files}"
+[ -f "${image}/config.sh"  ] && install -m 644 -p "${image}/config.sh" "${obs_files}"
```

Comment:
Should this one be 0755?

Reply:
existing files were predominantly 644, so I assume it doesn't matter. I made it 755 now.